src/AaveV2EthereumRatesUpdates_20230627/AaveV2EthereumRatesUpdates_20230627Test.t.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {ProtocolV2TestBase, ReserveConfig} from 'aave-helpers/ProtocolV2TestBase.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV2EthereumRatesUpdates_20230627} from './AaveV2EthereumRatesUpdates_20230627.sol';
+
+contract AaveV2EthereumRatesUpdates_20230627_Test is ProtocolV2TestBase {
+  AaveV2EthereumRatesUpdates_20230627 public proposalPayload;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 17564111);
+  }
+
+  function testPayload() public {
+    ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot(
+      'preTestAaveV2EthereumRatesUpdates_20230627',
+      AaveV2Ethereum.POOL
+    );
+
+    // 2. create payload
+    proposalPayload = new AaveV2EthereumRatesUpdates_20230627();
+
+    // 3. execute payload
+
+    GovHelpers.executePayload(vm, address(proposalPayload), AaveGovernanceV2.SHORT_EXECUTOR);
+
+    ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot(
+      'postTestAaveV2EthereumRatesUpdates_20230627',
+      AaveV2Ethereum.POOL
+    );
+
+    diffReports(
+      'preTestAaveV2EthereumRatesUpdates_20230627',
+      'postTestAaveV2EthereumRatesUpdates_20230627'
+    );
+
+    address[] memory assetsChanged = new address[](18);
+    assetsChanged[0] = AaveV2EthereumAssets.DAI_UNDERLYING;
```

Comment:
can you do instead 

    address[] memory assetsChanged = [DAI_UNDERLYING, FRAX_UNDERLYING ...] ?

Reply:
no